gt-opengis/pom.xml

```diff
@@ -48,7 +52,7 @@
 			<plugin>
 				<groupId>org.apache.felix</groupId>
 				<artifactId>maven-bundle-plugin</artifactId>
-				<version>3.5.0</version>
+				<version>4.2.1</version>
```

Comment:
❔General question: do you think the maven-bundle-plugin upgrade is worth applying everywhere? It seems like ddf-parent had some downsides to it, but it would be nice to make an upgrade like this and apply it to all modules as a default. If there edge cases where it causes problems then downgrade in those respective modules.

Reply:
I definitely think we should explore upgrading this plugin everywhere. These bundles all did seem to come up much faster after the upgrade. It'd be worth investigating. Really, we should check out all of the plugins we are using and update them all.